ci/docker/base/Dockerfile

```diff
@@ -26,7 +28,8 @@ RUN curl -L https://github.com/docker/compose/releases/download/$DOCKER_COMPOSE_
 
 # Final image
 # buster is for debian 10. the same version as firmware team has in its Dockerfile.
-FROM node:14-buster
+## Use a proxy or fallback to no proxy at all (direct access to Docker Hub).
```

Comment:
Imo no need to repeat the comment here. I'd only put it above the `CI_DOCKER_PROXY` declaration where the mentioned fallback actually happens. (It's the `=""` part.)